com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/IEvaluationProvider.java

```diff
@@ -55,6 +55,7 @@
      */
     CompletableFuture<Value> evaluateForBreakpoint(IBreakpoint breakpoint, ThreadReference thread, Map<IBreakpoint, Object> breakpointExpressionMap);
 
+    CompletableFuture<Value> evaluateForLogpoint(IBreakpoint breakpoint, ThreadReference thread, Map<IBreakpoint, Object> breakpointExpressionMap);
```

Comment:
The design here is weird. You can already design the interface of IBreakpoint for both types, yet provide two API on IEvaludationProvider, which is contradict to the interface usage.